cmd/release/main.go

```diff
@@ -57,6 +58,7 @@ type PluginRelease struct {
 	PluginVersion    string        `json:"version"`           // version of the plugin (including 'v' prefix)
 	PluginZipDigest  string        `json:"zip_digest"`        // <digest-type>:<digest> for plugin .zip download
 	PluginYAMLDigest string        `json:"yaml_digest"`       // <digest-type>:<digest> for buf.plugin.yaml
+	ImageDigest      string        `json:"image_digest"`      // <digest-type>:<digest> for Docker image
```

Comment:
What is the difference between `ImageDigest` and `GHCRImageDigest` ?

Reply:
`GHCRImageDigest` is the digest used to pull the image from ghcr.io registry and `ImageDigest` is the digest for the config.json of the image - this can be used to refer to the image after running `docker load`, or with `docker image inspect <id>`, and is shown in the `docker images` output.

Here's an example where `GHCRImageDigest=sha256:7634b9b51a6f7a49837f9ef6ca6acdb0804455b82b70e93b94eedee4c4b0112f` and `ImageDigest=sha256:df970372bba3f6c0f11f12f4b0675f2cb393763290291fd5d82e2527d0684113`:
```
# Fetch using GHCRImageDigest
$ docker pull ghcr.io/bufbuild/plugins-bufbuild-es@sha256:7634b9b51a6f7a49837f9ef6ca6acdb0804455b82b70e93b94eedee4c4b0112f
# Get ImageDigest from container
$ docker inspect ghcr.io/bufbuild/plugins-bufbuild-es@sha256:7634b9b51a6f7a49837f9ef6ca6acdb0804455b82b70e93b94eedee4c4b0112f | jq -r '.[].Id'
sha256:df970372bba3f6c0f11f12f4b0675f2cb393763290291fd5d82e2527d0684113
# Docker images output shows both GHCRImageDigest and (short) ImageDigest
$ docker images --digests | grep df970372
ghcr.io/bufbuild/plugins-bufbuild-es                    <none>                         sha256:7634b9b51a6f7a49837f9ef6ca6acdb0804455b82b70e93b94eedee4c4b0112f   df970372bba3   5 days ago      225MB
# Saving an image works with either ghcr.io/bufbuild/plugins-bufbuild-es@<GHCRImageDigest> or <ImageDigest>
$ docker save -o image.tar sha256:df970372bba3f6c0f11f12f4b0675f2cb393763290291fd5d82e2527d0684113
$ tar ztf image.tar
1d5104b3f8a8674d05100473f9dc3c7a09ce1c6d48c625a727797059b25e291e/
1d5104b3f8a8674d05100473f9dc3c7a09ce1c6d48c625a727797059b25e291e/VERSION
1d5104b3f8a8674d05100473f9dc3c7a09ce1c6d48c625a727797059b25e291e/json
1d5104b3f8a8674d05100473f9dc3c7a09ce1c6d48c625a727797059b25e291e/layer.tar
331eda04ad93d94b8339faf570c7e305fd7ce0c0178c17c0235ca68046a87f79/
331eda04ad93d94b8339faf570c7e305fd7ce0c0178c17c0235ca68046a87f79/VERSION
331eda04ad93d94b8339faf570c7e305fd7ce0c0178c17c0235ca68046a87f79/json
331eda04ad93d94b8339faf570c7e305fd7ce0c0178c17c0235ca68046a87f79/layer.tar
6bd868c4d3ee016c9d112ae1129f711488b56dbda2df9757d6b52ad4be7ce54b/
6bd868c4d3ee016c9d112ae1129f711488b56dbda2df9757d6b52ad4be7ce54b/VERSION
6bd868c4d3ee016c9d112ae1129f711488b56dbda2df9757d6b52ad4be7ce54b/json
6bd868c4d3ee016c9d112ae1129f711488b56dbda2df9757d6b52ad4be7ce54b/layer.tar
8cb43270873e01378aa08e270414eead0ec28c047541776cd4975074887b1ec0/
8cb43270873e01378aa08e270414eead0ec28c047541776cd4975074887b1ec0/VERSION
8cb43270873e01378aa08e270414eead0ec28c047541776cd4975074887b1ec0/json
8cb43270873e01378aa08e270414eead0ec28c047541776cd4975074887b1ec0/layer.tar
9cc0398bda22792e89701dcbf11b8111072e1910c7de1b81c0449eac4633919f/
9cc0398bda22792e89701dcbf11b8111072e1910c7de1b81c0449eac4633919f/VERSION
9cc0398bda22792e89701dcbf11b8111072e1910c7de1b81c0449eac4633919f/json
9cc0398bda22792e89701dcbf11b8111072e1910c7de1b81c0449eac4633919f/layer.tar
df970372bba3f6c0f11f12f4b0675f2cb393763290291fd5d82e2527d0684113.json
manifest.json
```

Reply:
When you say `ImageDigest`, this is the ImageID of the config?

https://github.com/opencontainers/image-spec/blob/main/config.md#imageid

And the `GHCRImageDigest` is the manifest digest, this is the one you use in docker pull, etc. ?

Reply:
Suggestion, rename this to `ConfigImageID` to be consistent with the "spec".

I think when folks thing of "image digest" it's actually the manifest digest, this naming is just the worst. I'm sorry.